xml/admin_nfsganesha.xml

```diff
@@ -389,6 +405,80 @@ EXPORT {
     </variablelist>
    </sect3>
   </sect2>
+
+  <sect2 xml:id="ganesha-getting-exports">
+   <title>Getting Exports Configuration</title>
+   <para>
+    To get the exports configuration details, follow these steps:
+   </para>
+   <procedure>
+    <step>
+     <para>
+      Get the &rados; pool name and namespace for &ganesha; exports. The
+      following command outputs a string of the
+      <replaceable>POOL_NAME</replaceable>/<replaceable>NAMESPACE</replaceable>
+      form.
+     </para>
+<screen>&prompt.cephuser;ceph dashboard get-ganesha-clusters-rados-pool-namespace
+cephfs_data/ganesha</screen>
+    </step>
+    <step>
+     <para>
+      Given the pool name and namespace, list the &rados; objects available on
```

Comment:
"Given the pool name and namespace"... as in, you've received it from the above command? If so,

"Now you have the pool name and namespace, list the ..."